cola/ops/operators.py

```diff
@@ -102,6 +106,10 @@ def __str__(self):
     def _matmat(self, X):
         return X
 
+    def to(self, device):
```

Comment:
why is it different for scalarmul?

Reply:
@mfinzi I explained this changed to you before. Having the identity be part of `ScalarMul` was making the `to()` function behavior become really complicated. It's much easier and readable to have the Identity be a noop